kyuubi-server/src/main/antlr4/org/apache/kyuubi/sql/KyuubiSqlBaseParser.g4

```diff
@@ -24,8 +24,11 @@ singleStatement
     ;
 
 statement
-    : (KYUUBI | KYUUBIADMIN) runnableCommand                        #runnable
-    | .*?                                                           #passThrough
+    : (KYUUBI | KYUUBIADMIN) runnableCommand                                                  #runnable
+    | SELECT TABLE_SCHEM COMMA TABLE_CATALOG FROM SYSTEM_JDBC_SCHEMAS
+      (WHERE (TABLE_CATALOG EQ catalog=STRING+)? AND? (TABLE_SCHEM LIKE schema=STRING+)?)?
+      (ORDER BY TABLE_CATALOG COMMA TABLE_SCHEM)?                                             #getSchemas
```

Comment:
This seems to be a required field

Reply:
not sure what field you refered ?

Reply:
do you mean `ORDER BY` ? yeah, it seems to be hard code in Trino-driver.